docs/features/jib.md

```diff
@@ -0,0 +1,19 @@
+# Using Jib
+
+[Jib](https://github.com/GoogleContainerTools/jib/tree/master/jib-core) is a library for building Docker images.
+
+In order to use it along with Testcontainers, a DockerClient is needed.
+
+<!--codeinclude-->
+[DockerClient](../../core/src/test/java/org/testcontainers/containers/JibTest.java) inside_block:dockerClientInstance
+<!--/codeinclude-->
+
+The DockerClient should be used as follows
+
+<!--codeinclude-->
+[Jib#from(String)](../../core/src/test/java/org/testcontainers/containers/JibTest.java) inside_block:jibContainer1
+[Jib#from(DockerClient, DockerDaemonImage)](../../core/src/test/java/org/testcontainers/containers/JibTest.java) inside_block:jibContainer2
```

Comment:
Let's add a documentation paragraph, how the second example needs to be used if the Testcontainers `DockerClient` should perform the image pull. And in which cases is this required?

Reply:
I have added a description for both examples and pointing to Jib's documentation